src/to-have-style.js

```diff
@@ -37,7 +38,7 @@ function printoutStyles(styles) {
 // received computed styles
 function expectedDiff(expected, computedStyles) {
   const received = Array.from(computedStyles)
-    .filter(prop => expected[prop])
+    .filter(prop => expected[prop] !== undefined)
```

Comment:
This is needed to get a proper diff for `expect(something).toHaveStyle({width: ''})` (expect width to be empty). Without this change it prints "the diff has no visual difference"